### ISSUE TITLE
Cannot build on fresh install

### ISSUE THREAD

**Issue by tim-tim707** [everalert/annodue#7]
No idea if its Zig related:
```sh
C:\Users\Tim\Desktop\STAR WARS RACER DIR\annodue> zig build release -Doptimize=ReleaseSafe -Dver="0.0.1" -Dminver="0.0.0" -Drop="C:\Users\Tim\Desktop\STAR WARS RACER DIR\annodue\release"
C:\Users\Tim\Desktop\STAR WARS RACER DIR\annodue\build.zig.zon:1:2: error: missing top-level 'paths' field
.{ [1] Fetch Packages
 ^
Compile Build Script
```

Found [this](https://github.com/ziglang/zig/issues/17491) related in the zig project, but there isn't a .path variable in build.zig.zon.

Tried `zig fetch someURL` but got hit with another error:
```sh
zig fetch https://github.com/ziglibs/ini/archive/2b11e8fef86d0eefb225156e695be1c1d5c35cbc.tar.gz
error: unable to connect to server: TlsInitializationFailed
Fetch
```

Should I be on some specific zig version ? The "unable to connect to server" issue is strange since my internet connection works fine

Edit: Tried from WSL with a linux zig install only to have the same issue. I suspect more it is a zig version

**Comment by everalert** [comment]
Hallo, I'm not sure if it's related to your issue, but you need Zig 0.11.0.

**Comment by tim-tim707** [comment]
Fixed using zig 0.11.0 and `zig-windows-x86_64-0.11.0` specifically !
